src/models/atmosphere/FGStandardAtmosphere.cpp

```diff
@@ -126,6 +130,15 @@ bool FGStandardAtmosphere::InitModel(void)
   TemperatureBias = 0.0;
   CalculateLapseRates();
   CalculatePressureBreakpoints();
+
+  // Density altitude parameters
+  // Density altitude formula only valid up until top of the Troposhere
+  TroposphereMaxAltitude = GeometricAltitude((*StdAtmosTemperatureTable)(2, 0));
+  // Standard sea level temp / Troposphere lapse rate
+  DATroposphereFactor = -(*StdAtmosTemperatureTable)(1, 1) / LapseRateVector[0];
+  // Unitless exponent computed using SI values from LR/(g0M - L*R) 
+  DATroposphereExponent = 0.2349781324440659;  
```

Comment:
You made `DATroposphereFactor` depend on the value of `LapseRateVector[0]` but hardcoded the exponent `DATroposphereExponent` while its depends also of `LapseRateVector[0]` ?

Reply:
Yep, see the discussion I had with Dennis with regards to the exponent on issue [44](https://github.com/JSBSim-Team/jsbsim/issues/44)